MeetU/MeetU/MeetU.csproj

```diff
@@ -262,6 +262,7 @@
     <Content Include="Content\bootstrap-theme.min.css.map" />
     <Content Include="Content\bootstrap-theme.css.map" />
     <None Include="Content\hello-world.less" />
+    <Content Include="Content\Spinner.less" />
```

Comment:
yes I think I forgot to add the new file. I thought git commit -am"" would do that for me.